### PR TITLE
Add Computer Alliance [Retail]

### PR DESCRIPTION
**Pull request by Jawshy**
#### Changes:
- Added: [Computer Alliance](http://www.computeralliance.com.au/) under retail. 
  - No TFA. No Twitter handle.

### CODE REVIEW

File: _data/retail.yml [hunk 1]
```diff
@@ -31,6 +31,11 @@ websites:
       tfa: No
       twitter: comixology
 
+    - name: Computer Alliance
+      url: http://www.computeralliance.com.au/
+      img: computeralliance.png
+      tfa: No
+
```
Comment (Carlgo11) [inline _data/retail.yml:45]: I suggest `facebook: ComputerAllianceAus` as their Twitter doesn't seem to be monitored by them.

Comment (Jawshy) [inline _data/retail.yml:45]: Added.

**Comment by Carlgo11** [comment]
Again, by the looks of it I wouldn't classify it as a significant site.
Plus, they could start with supporting TLS before 2FA 😜

**Comment by Jawshy** [comment]
> Plus, they could start with supporting TLS before 2FA :stuck_out_tongue_winking_eye:

Damn right! 
[DAMN RIGHT !](https://www.youtube.com/watch?v=gHQhlhz-_K0)

**Pull request closed without merging**